### PR TITLE
audio paste handler

### PR DESCRIPTION
**Pull request by adrianjost**
image

### CODE REVIEW

File: src/plugins/url-handler/index.js [hunk 1]
```diff
@@ -27,9 +34,20 @@ const HandlePaste = {
       console.log('indeed a url')
 
       if (YoutubeHandler().validate(text)) {
+        console.log('for Youtube')
```
Comment (janrenz) [inline src/plugins/url-handler/index.js]: remove logs or use a logger that only logs in dev,,

**Comment by FWirtz** [comment]
looks cool!
I can only review this on Monday, but I'll have a look then.

Also, please change the base to `development` and not `master`, I would love to keep [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) going 😊

**Comment by FWirtz** [comment]
@adrianjost Thank you so much for the work, finally had the chance to take a look at it!

I needed to do some refactoring as I introduced some breaking changes in the `development` branch but now managed to merge your work within #107.

I also changed the `preload` to `metadata` instead of `none`, as this also fetches the preview image.

THANKS!!! ❤️👏

**Comment by adrianjost** [comment]
It's been a pleasure ;D
and looks like you're faster than me. 👍 
I was about to change everything you mentioned right now.

**Pull request closed without merging**